### DISCUSSION TITLE
GitHub Support does not respond even after 2 months

### DISCUSSION THREAD

**Discussion by Beo-Coder** [community/community#68449] [GitHub Education] [answered]
I contacted GitHub support over 2 months ago and still haven't heard back. I have asked several times if there is any news etc. about my case but never received an answer.
Since this is a problem that only GitHub can help me with, I have to rely on their support.

Is it normal that this takes so long?
I have never had contact with the support of GitHub before and therefore have no experience with it.

**Comment by samus-aran** [comment, marked as answer]
Hi @Beo-Coder  👋 

We're sorry to hear of your frustration. We are seeing a higher volume of student developer pack inquiries coming through. We are working our way through addressing these as fast as possible. 

I can share that your request has been passed onto the GitHub Education team and the internal team will investigate the School details, but I am afraid I do not have a timeline to share for when you can expect this to happen. When I have an update I will either update here in the through or through your tickets.

**Reply by Beo-Coder** [reply]
Thanks for info.
I was afraid that my ticket has gone down.
But thanks for the quick reply

**Comment by chrisstef** [comment]
I am disappointed. It's been 1 month after my account was flagged as spam, for no obvious reason, and this has already costed me 2 job applications where I couldn't showcase my codebase and give access to a 3rd party app required for the application process. Github support hasn't reached out to me yet.

**Reply by adxthya** [reply]
Did you get your account back?

**Reply by saahilkhanna** [reply]
Same thing has happened to me. Ridiculous how long it takes they say 7 days but it has been two months already.

**Comment by ozturkyusufziya** [comment]
I've been waiting for a response for 1 month. A week ago, I received a message saying "We have not forgotten about you..." but now I feel like I have been forgotten. From now on, I'll be tagging @github-staff in every discussion that has this issue. I guess you need to be Karen to get a problem solved.

**Reply by saahilkhanna** [reply]
@github-staff has done nothing for me either,absolutely ridiculous

**Comment by hutaoshusband** [comment]
Hello. @github-staff. I am also not getting any response, while already waiting for over a month for a response. Please help me, it is pretty important. #4424445

**Comment by elizawonggu** [comment]
It definitely seems frustrating, especially when the issue affects your work or studies. Waiting that long for support is a lot like relying on [professional catering services](https://pearllemoncatering.com/) clear communication and timely updates make a huge difference in the overall experience.
